packages/validate/src/lib/factory/group.js

```diff
@@ -23,5 +24,5 @@ export const addGroup = Store => nodes => {
  * 
  */
 export const removeGroup = Store => groupName => {
-    Store.dispatch(ACTIONS.REMOVE_GROUP, groupName);
+    Store.dispatch(ACTIONS.REMOVE_GROUP, groupName, [ clearError(groupName) ]);
```

Comment:
If removeGroup changed to 
```
clearError(groupName)(Store.getState());
Store.dispatch(ACTIONS.REMOVE_GROUP, groupName);
```
then the error is removed before the group is removed and we wouldn't have to change the clearError function at all.

I think that's a bit cleaner, especially since clearError is already a pile of side effects.

Reply:
Updated in new push.  Thanks!